README.md

```diff
@@ -19,19 +19,36 @@ Splunk Connect for Kafka is a Kafka Connect Sink for Splunk with the following f
 
 1. Clone the repo from https://github.com/splunk/kafka-connect-splunk
 2. Verify that Java8 JRE or JDK is installed.
-3. Run `bash build.sh`. The build script will download all dependencies and build Splunk Connect for Kafka.
-
-Note: The resulting "splunk-kafka-connect*.tar.gz" package is self-contained. Bundled within it are the Kafka Connect framework, all 3rd party libraries, and Splunk Connect for Kafka.
+3. Run `mvn package`. This will build the jar in the /target directory. The name will be "splunk-kafka-connect-v*.jar".
```

Comment:
the v* doesn't look so great.
can we try [VERSION] or (version) to represent this better

Reply:
Went with [VERSION]